src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -40,7 +36,7 @@ public static String convertWordToPigLatin(String word) {
 	
 	
 	public static String convertStringToPigLatin(String inputString) {	
-		String[] words = inputString.split(" ");
+		String[] words = inputString.split("\\W+");
```

Comment:
I have currently extended this solution into a full interpunctuation handler in a separate branch. This means that this pull request is outdated.